### DISCUSSION TITLE
安装的3.0.1集群班 数据源中心登记mysql数据源一直报jdbc链接错误

### DISCUSSION THREAD

**Discussion by opc-x** [apache/dolphinscheduler#13864] [Q&A]
确认链接信息没错 网络也通，连接信息用linux命名行客户端可以连通，但是用ds始终不同 报jdbc连接错误 从官网换了不同的驱动包 放到api-server/lib下 以及worker-server/lib下都未能成功

**Comment by WilliamChen-luckbob** [comment]
官方文档的集群启动就是有问题的，你还需要在对应的项目下（比如/worker-server/conf/application.yaml）中配置你的数据源，跟你的jar包大概率是没关系，另外，按官方文档的步骤，数据源初始化脚本也启动不了的，你还得传jar包到tools/lib中才能执行

**Comment by anlei-cdh** [comment]
看看worker节点是 application-mysql.yaml  中的连接数据库的地址是否正确。
